cmd/repo.go

```diff
@@ -33,6 +35,22 @@ var addCmd = &cobra.Command{
 	Run: func(cmd *cobra.Command, args []string) {
 		spinner, _ := yacspin.New(util.SpinnerConf)
 
+		_, err := os.Stat(util.RepoPath)
+		if err != nil {
```

Comment:
I think it's better for the code quality if you do `os.IsNotExist(err)` and then do a `else if err != nil` instead of the current way